catalyst/contrib/dl/runner/nlp/huggingface_runner.py

```diff
@@ -0,0 +1,19 @@
+from catalyst.dl import Runner
+from catalyst.dl.utils import is_wrapped_with_ddp
+
+
+class HuggingfaceRunner(Runner):
+    """
+    @TODO
+    """
+
+    def _handle_batch(self, batch):
+        # output is always tuple in hf
+        # also if model is sepcified with task
+        # loss is stored in outpt[0]
+        loss = self.model(**batch)[0]
+
+        if is_wrapped_with_ddp(self.model):
+            loss = loss.mean()
```

Comment:
not sure if we need it, Catalyst already have `distributed_mean` for metrics

Reply:
I get loss with shape (2,) if model is DataParallel, so I can't do backward. Is this a bug?

Reply:
I get output shape of (2,) if i use DataParallel. So I need reduction because I obviously can't do backward. Is it a bug?

Reply:
looks like so (not sure about huggingface internals)